round4/TabsOFF.md

```diff
@@ -0,0 +1,123 @@
+5. As an academic researcher, I often open tabs related to various research papers and resources. I want TabsOFF to automatically save the URLs of long-unused tabs to a specified document before closing them, ensuring I don't lose important references.
+6. As a user concerned with privacy, I want TabsOFF to offer a feature that automatically closes tabs containing sensitive information if they've been left unattended for a certain period, helping me safeguard my personal data.
+7. As a productivity enthusiast, I want to track the amount of time I spend on different websites through TabsOFF, enabling me to identify and reduce time spent on unproductive or distracting sites.
+8. As a stay-at-home parent managing household tasks and online activities, I want TabsOFF to remind me to review tabs related to shopping, recipes, and family activities so that I can stay organized and efficient in managing my daily responsibilities.
+9. As a professional working from home, I want TabsOFF to automatically close tabs that I haven't accessed in over 72 hours, except for those I've marked as important, so my browser stays clean and efficient without manual intervention.
+10. As a financial analyst conducting research on market trends, I want TabsOFF to provide insights into my browsing habits and suggest ways to improve my tab organization so that I can make more informed investment decisions.
+11. As an individual conscious about browser performance, I want TabsOFF to provide statistics on how many tabs were closed and how much memory was freed up weekly, motivating me to maintain a cleaner browser environment.
+12. As a project manager constantly juggling tasks, I need TabsOFF to only remind me about non-essential tabs during my scheduled review times, so I can focus on critical work without distraction but still declutter my workspace at designated times.
+13. As a user concerned about losing track of important tabs, I want the option to set specific tabs as "Do Not Close," allowing me to keep essential tabs open indefinitely without them being automatically closed by TabsOFF.
+14. As someone who uses the browser for leisure reading, I'd like TabsOFF to suggest closing tabs related to articles or blogs if they've been open for more than a few days, helping me manage my leisure time more effectively.
+15. As a software developer researching different technologies, I want TabsOFF to save memory and improve browser performance by encouraging the closure of unnecessary tabs so that I can work more efficiently.
+
+
+## References & Inspiration
+[Arc browser](https://arc.net/)
+
+## Technical Details
+### User Interface
+Key Elements:
+Pop-Up Dashboard: Quick overview of unused tabs categorized by the last accessed time.
+
+Sidebar for Detailed Organization: A collapsible sidebar that categorizes tabs more extensively.
```

Comment:
how would this work if people already use the chrome tab groups for other things? would you have tab groups within those already grouped tabs or will it be separate? how would you notify people in this case as well

Reply:
We plan to have our own grouping mechanism in the dashboard/popup that will not be related to the google's tab grouping options. Our grouping will be by time/day thresholds.